### PR TITLE
{lib}[GCC/10.3.0] Boost.Python v1.76.0 w/ Python 3.9.5

### PR DESCRIPTION
**Pull request by SIB-software**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/b/Boost.Python/Boost.Python-1.76.0-GCC-10.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+source_urls = ['https://boostorg.jfrog.io/artifactory/main/release/%(version)s/source/']
+sources = ['boost_%s.tar.gz' % '_'.join(version.split('.'))]
+patches = ['Boost-1.71.0_fix-Python3.patch']
+checksums = [
+    '7bd7ddceec1a1dfdcbdb3e609b60d01739c38390a5f956385a12f3122049f0ca',  # boost_1_76_0.tar.gz
+    '60e3aede2f444a3855f4efed94d1de5c2887983876e0fae21f6ca5cfdc53ea96',  # Boost-1.71.0_fix-Python3.patch
+]
+
+multi_deps = {'Python': ['3.9.5']}
```
Comment (verdurin) [inline easybuild/easyconfigs/b/Boost.Python/Boost.Python-1.76.0-GCC-10.3.0.eb:21]: @SIB-software For `Python` please specify it as a simple dependency. For the newer toolchains we no longer include a `Python 2.x` option (with rare exceptions).

**Review by verdurin — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/b/Boost.Python/Boost.Python-1.76.0-GCC-10.3.0.eb (hunk 1 above)
Comment (smoretti) [reply easybuild/easyconfigs/b/Boost.Python/Boost.Python-1.76.0-GCC-10.3.0.eb]: done

Comment (verdurin) [reply easybuild/easyconfigs/b/Boost.Python/Boost.Python-1.76.0-GCC-10.3.0.eb]: Thanks.

**Review by verdurin — APPROVED** [review]
Looks fine.

**Comment by verdurin** [comment]
@boegelbot please test @ generoso

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
easybuild-c7.novalocal - Linux centos linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/8ef9580560af660d4d9cde59302bfd20 for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
nuc.lan - Linux Fedora 33, x86_64, Intel(R) Core(TM) i7-8650U CPU @ 1.90GHz, Python 3.9.6
See https://gist.github.com/e37795ae74cc04801be33ddcc805e799 for a full test report.

**Comment by verdurin** [comment]
Going in, thanks @SIB-software!

**Pull request merged**